Cubical/Categories/Colimits.agda

```diff
@@ -0,0 +1,4 @@
+{-# OPTIONS --safe #-}
+module Cubical.Categories.Colimits where
```

Comment:
I would not add another subpackage for colimits but just have them all in limits. A colimit is just a form of limit after all